docker-compose.yml

```diff
@@ -181,6 +181,21 @@ services:
       context: .
       dockerfile: integration/hdfs/Dockerfile
 
+  pandas-master:
+    # Usage:
+    #   export PYTHON_VERSION=3.6
+    #   docker-compose build cpp
+    #   docker-compose build python
+    #   docker-compose build pandas-master
```

Comment:
Can you add the necessary command line options to do a no-cache-rebuild of the pandas-master image to get the real latest pandas commit, not the one that was cached by docker?

Reply:
Sure, good point!